app/code/Magento/Catalog/Model/Category.php

```diff
@@ -664,7 +664,8 @@ public function getImageUrl($attributeCode = 'image')
             if (is_string($image)) {
                 $store = $this->_storeManager->getStore();
 
-                $isRelativeUrl = substr($image, 0, 1) === '/';
+                $imagePath = \Magento\Framework\UrlInterface::URL_TYPE_MEDIA;
+                $isRelativeUrl = substr($image, 1, strlen($imagePath)) === $imagePath;
 
                 $mediaBaseUrl = $store->getBaseUrl(
                     \Magento\Framework\UrlInterface::URL_TYPE_MEDIA
```

Comment:
Duplicated `\Magento\Framework\UrlInterface::URL_TYPE_MEDIA` instance, 
since you have loaded it before, just pass the variable here, like:
```
$mediaBaseUrl = $store->getBaseUrl($imagePath);
```

Reply:
Fixed